test/parser/test-grub2-blscfg-opts-grubenv.c

```diff
@@ -21,7 +21,7 @@ void run_test(struct parser_test *test)
 			     "/loader/entries/6c063c8e48904f2684abde8eea303f41-4.15.2-302.fc28.x86_64.conf",
 			     "title Fedora (4.15.2-302.fc28.x86_64) 28 (Twenty Eight)\n"
 			     "linux /vmlinuz-4.15.2-302.fc28.x86_64\n"
-			     "initrd /initramfs-4.15.2-302.fc28.x86_64.img\n"
+			     "initrd $blank /initramfs-4.15.2-302.fc28.x86_64.img $blank\n"
```

Comment:
This might warrant its own test rather than adding into the env test here.

Reply:
That's true, it could be spun into a whole new test, but I think of it as an edge case of what's already being tested there with `$kernelopts`; variable expansion, but a case where the variable is blank.